docker-entrypoint.sh

```diff
@@ -44,6 +44,13 @@ if [[ "$1" == "kong" ]]; then
   if [[ "$2" == "docker-start" ]]; then
     kong prepare -p "$PREFIX" "$@"
 
+    # remove all dangling sockets in $PREFIX dir before starting Kong
```

Comment:
Having leftover sockets is not something that happens under normal circumstances, so I think we should log a warning here.

Something like...

```sh
LOGGED_SOCKET_WARNING=0

if [[ -S $localfile ]]; then
  if (( LOGGED_SOCKET_WARNING == 0 )); then
    printf >&2 'WARN: found dangling unix sockets in the prefix directory '
    printf >&2 '(%q) ' "$PREFIX"
    printf >&2 'while preparing to start Kong. This may be a sign that Kong '
    printf >&2 'was previously shut down uncleanly or is in an unknown state '
    printf >&2 'and could require further investigation.\n'

    LOGGED_SOCKET_WARNING=1
  fi

  # 1. using the verbose (-v) flag
  # 2. explicitly sending output to stderr
  rm -fv "$localfile" >&2
fi
```

The warning here is copy/pasted from [kong start](https://github.com/Kong/kong/blob/d7e3cbc7ac2f28fa0cc057859203b01e61f993f7/kong/cmd/start.lua#L30-L34)

Reply:
Good idea